### PR TITLE
Fix : Toc heading not working with this `…` dot

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

Ticket - https://dashboard.helpninja.com/#/dashboard/5ce7c4862b84d8603423dbc8/threads/437946
SS - https://drive.google.com/file/d/1yUcztCHq_3_vfBuV1Ekxy9qUesFzPI-I/view
Trello - https://trello.com/c/dKGJ2XIi/146-fix-toc-heading-not-working-with-this-dot

### ScreenShot
- https://share.getcloudapp.com/DOu95Wpo
- https://share.getcloudapp.com/4guOjgE7

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Use heading with more than 2 dots it will not work in front-end [master]
- Switch to this use heading with multiple dots it will work [toc-dot-issue]

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by vrundakansara — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -162,7 +162,7 @@ When you use the Ultimate Addons for Gutenberg along with the free Astra theme,
 
 ### 1.19.0 ###
 * Improvement: Ability to modify breakpoints for Dynamic CSS.
-* Improvement: Table of Content - Exclude specific headings from Table of Contents block.
```
Comment (vrundakansara) [inline README.md:165]: Are we supposed to remove this changelog?

Comment (AbhijitNage123) [reply README.md:165]: I'm sorry. Updated

File: assets/js/table-of-contents.js [hunk 2]
```diff
@@ -14,6 +14,7 @@
 		}
 
 		var parsedSlug = slug.toString().toLowerCase()
```
Comment (vrundakansara) [inline assets/js/table-of-contents.js:16]: We have a similar function in edit.js too.
Do we need to change there too?

Comment (AbhijitNage123) [reply assets/js/table-of-contents.js:16]: Right. we are creating header from edit.js so we need to update it too.

**Review by vrundakansara — APPROVED** [review]

**Pull request merged**